### PR TITLE
Add some states related to Somfy Thermostat

### PR DESCRIPTION
**Pull request by tetienne**

### CODE REVIEW

**Comment by vlebourl** [comment]
Could you also add commands and states related to away and boost mode in water heater? 
cf https://github.com/iMicknl/ha-tahoma/pull/688/files
```py
core:BoostModeDurationState
io:AwayModeDurationState
setBoostModeDuration
setAwayModeDuration
```

File: pyoverkiz/enums/state.py [hunk 1]
```diff
@@ -14,15 +14,16 @@ class OverkizAttribute(str, Enum):
 class OverkizState(str, Enum):
     """Device states used by Overkiz."""
 
-    CORE_ACTIVE_ZONES = "core:ActiveZonesState "
+    CORE_ACTIVE_ZONES = "core:ActiveZonesState"
```
Comment (iMicknl) [inline pyoverkiz/enums/state.py:17]: Good catch!

Comment (iMicknl) [reply pyoverkiz/enums/state.py:17]: Can you maybe fix the ventilation mode as well? (in this PR or another one :D)

Comment (tetienne) [reply pyoverkiz/enums/state.py:17]: What’s the issue with`    IO_VENTILATION_MODE = "io:VentilationModeState"`
?

Comment (iMicknl) [reply pyoverkiz/enums/state.py:17]: SET_VENTILATION_MODE (command) has a typo in its enum

Comment (vlebourl) [reply pyoverkiz/enums/state.py:17]: there was a typo, but looks fixed already

Comment (iMicknl) [reply pyoverkiz/enums/state.py:17]: https://github.com/iMicknl/python-overkiz-api/blob/837f019b88d5f6c1f0e16ac981489d5ef87783d5/pyoverkiz/enums/command.py#L73

**Comment by tetienne** [comment]
@vlebourl done

**Review by vlebourl — APPROVED** [review]

**Pull request merged**